docs/user-guide/installation/windows.md

```diff
@@ -269,7 +269,7 @@ Clang does not work to compile on MSYS2 at this time. It has in the past but the
 HELICS can also be built with the standalone MinGW
 
 - We assume you have MinGW installed or know how to install it.
-- [Boost](https://www.boost.org/doc/libs/1_70_0/more/getting_started/windows.html); you can use the [Windows installer](https://dl.bintray.com/boostorg/release/1.70.0/binaries/) for Boost installed in the default location
+- [Boost](https://www.boost.org/doc/libs/1_75_0/more/getting_started/windows.html); you can use the [Windows installer](https://sourceforge.net/projects/boost/files/boost-binaries/) for Boost installed in the default location
```

Comment:
Would we want this at 1.76 now?

Reply:
I put it at 1.75 since I've tried building with 1.75 in more configurations than 1.76 (and there might be a small mental thing with it being right before 1.77 which had some changes that broke things).